topdown/cidr_test.go

```diff
@@ -44,3 +46,133 @@ func TestNetCIDRExpandCancellation(t *testing.T) {
 	}
 
 }
+
+func TestBuiltinNetCIDRMerge(t *testing.T) {
```

Comment:
I think with our newer externalized topdown test we should change these to be in https://github.com/open-policy-agent/opa/tree/master/test/cases/testdata (see https://github.com/open-policy-agent/opa/blob/master/test/cases/testdata/helloworld/test-helloworld-1.yaml for instructions)

Reply:
Tests updated to newer format.